rerun_py/Cargo.toml

```diff
@@ -81,6 +81,7 @@ comfy-table.workspace = true
 crossbeam.workspace = true
 datafusion.workspace = true
 datafusion-ffi.workspace = true
+datafusion-python.workspace = true
```

Comment:
this seems to be quite a large dependency for what's mostly a type annotation change? I figure us taking this on is inevitable though anyways?

Leaving this to @timsaucer how's literally the expert on this :)